### PR TITLE
[mono-2018-06] [llvm] Fix mxe+release_60 build

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

File: src/mono-runtimes/mono-runtimes.projitems [hunk 1]
```diff
@@ -159,7 +159,7 @@
 
     <_MonoCrossRuntime Include="cross-arm-win" Condition="$(AndroidSupportedTargetAotAbisForConditionalChecks.Contains (':win-armeabi-v7a:'))">
       <Strip>$(AndroidMxeFullPath)\bin\$(MingwCommandPrefix32)-strip</Strip>
-      <StripFlags>-S</StripFlags>
```
Comment (jonpryor) [inline src/mono-runtimes/mono-runtimes.projitems]: Do we not need `-S` because mono SDKs is now stripping outputs?

**Comment by jonpryor** [comment]
What is the *exact* issue which is being fixed?

File: src/mono-runtimes/mono-runtimes.projitems (hunk 1 above)
Comment (luhenry) [reply src/mono-runtimes/mono-runtimes.projitems]: No, `-S` triggers a SIGSEGV for some reasons, and the resulting binary is the same with and without the option.

**Comment by luhenry** [comment]
@jonpryor the issue is it just wouldn't build and link with the newer `llvm:release_60`. Also, note this PR is going into `xamarin-android:mono-2018-06` and not straight into `xamarin-android:master`.

**Pull request merged**